packages/terra-cell-grid/src/terra-dev-site/doc/example/CellGridBoth.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import CellGrid, {
+  Cell,
+} from 'terra-cell-grid';
+
+const CellGridBoth = () => (
+  <CellGrid
+    dividerStyle="both"
+  >
+    <Cell key="cell-0" width={{ static: { value: 80, unit: 'px' } }}>60px</Cell>
```

Comment:
In each of the cell-grid examples -- cell contents say "60px" and value is set to "80" px.
```suggestion
    <Cell key="cell-0" width={{ static: { value: 80, unit: 'px' } }}>80px</Cell>
```

Reply:
Updated.